projects/aca-content/src/lib/components/search/search-save/sidenav/save-search-sidenav.component.ts

```diff
@@ -42,10 +43,12 @@ export class SaveSearchSidenavComponent implements OnInit {
   savedSearchesService = inject(SavedSearchesService);
   appService = inject(AppService);
   translationService = inject(TranslationService);
+  userPreferenceService = inject(UserPreferencesService);
```

Comment:
I believe this service can be private because it's unused outside component and readonly as it's not reassigned